docs/Rules/ReservedParams.md

```diff
@@ -11,7 +11,7 @@ title: ReservedParams
 
 ## Description
 
-You cannot use reserved common parameters in an advanced function.
+You cannot use [reserved common parameters](https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_commonparameters) in an advanced function.
```

Comment:
Recommend using link-reference notation for maintainability - when we sync these docs to the Learn platform, we have to use site-relative URLs for anything on `learn.microsoft.com`. There will be a smaller diff between this file and the synced file if this uses reference links.

```suggestion
You cannot use [reserved common parameters][01] in an advanced function.

[01]: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_commonparameters
```

I can't suggest changes to non-edited lines, but generally it's preferable to add reference links to the bottom of a document.